docs/general/ad-filtering/create-own-filters.md

```diff
@@ -4627,7 +4627,7 @@ Open the browser console while on a web page to see the timing statistics for se
 
 **Always printed:**
 
-- `selectorParsed` — text of eventually parsed selector
+- `selectorParsed` — text of the parsed selector, i.e. may differ from the given one
```

Comment:
has been fixed in this PR: https://github.com/AdguardTeam/KnowledgeBase/pull/553/files#diff-190ba525f7c4d0b0eda33dab9df98df01d3ac072d6aeef951f044c96d58a25c9:~:text=%2D%20%60selectorParsed%60%20%E2%80%94%20text%20of%20the%20parsed%20selector%2C%20may%20differ%20from%20the%20input%20one